### ISSUE TITLE
Renaming lua functions crashes Ctrlr

### ISSUE THREAD

**Issue by keinstein** [RomanKubiak/ctrlr#167]
Hi,

Ctrlr seems not to be fault tolerant. I came accross several situations when it simply crashed.
One situation occurs when I have typo in the function name of a lua function. When I correct the typo, the function name gets not updated in the GUI and the GUI offers no option to update the function name. Finally the GUI does not allow to select the new function name and does not complain. It simply crashes when the function is called.

**Comment by dobo365** [comment]
Yep, there are obvious things that you should avoid doing even of there are basic ones
We all learn the hard way  ;-)
The only way to recover from this is to edit the .panel file in Notepad++ and of course to remember what you did.
Good luck with your panel building!

**Comment by keinstein** [comment]
Or to fix Ctrlr.
